### PR TITLE
8298225: [AIX] Disable PPC64LE continuations on AIX

### PR DESCRIPTION
**Pull request by backwaterred**
This small change adds an import to the generated ad_ppc.cpp file to allow it to find Contiuations::enabled, and sets VMContinuations to false on AIX.

---------
### Progress
- [ ] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8298225](https://bugs.openjdk.org/browse/JDK-8298225): [AIX] Disable PPC64LE continuations on AIX

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/11550/head:pull/11550` \
`$ git checkout pull/11550`

Update a local copy of the PR: \
`$ git checkout pull/11550` \
`$ git pull https://git.openjdk.org/jdk pull/11550/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 11550`

View PR using the GUI difftool: \
`$ git pr show -t 11550`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/11550.diff">https://git.openjdk.org/jdk/pull/11550.diff</a>

</details>

### CODE REVIEW

File: src/hotspot/share/adlc/main.cpp [hunk 1]
```diff
@@ -229,6 +229,7 @@ int main(int argc, char *argv[])
   AD.addInclude(AD._CPP_file, "opto/regalloc.hpp");
   AD.addInclude(AD._CPP_file, "opto/regmask.hpp");
   AD.addInclude(AD._CPP_file, "opto/runtime.hpp");
+  AD.addInclude(AD._CPP_file, "runtime/continuation.hpp");
```
Comment (TheRealMDoerr) [inline src/hotspot/share/adlc/main.cpp:232]: This adds it for all platforms. Isn't it sufficient to add it in `source %{` section of the ad file?

**Review by TheRealMDoerr — COMMENTED** [review]
Mainline will get forked today. Please create a new PR in the JDK 20 stabilization
repository once it's available. This one get get closed afterwards.

**Comment by TheRealMDoerr** [comment]
https://github.com/openjdk/jdk20 is now open for P3 bug fixes. (They will get merged into this repo after some time.)

**Comment by backwaterred** [comment]
> https://github.com/openjdk/jdk20 is now open for P3 bug fixes. (They will get merged into this repo after some time.)

Thanks. I am closing this PR, and will re-open one on jdk20 after learning if the ppc.ad modification is feasible. The build is looking good so far, so I suspect it was my error that kept it from working the first time.

**Pull request closed without merging**